css/styles.css

```diff
@@ -0,0 +1,299 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --white: #FFFFFF;
+  --black: #1C1C1C;
+  --primary-gradient: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: var(--white);
+  font-size: 1.6rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 70px;
+}
+
+nav a {
+  color: var(--black);
+  text-decoration: none;
+  font-weight: 300;
+}
+
+.navigation {
+  list-style: none;
+  display: flex;
+  gap: 37px;
+}
+
+.planIT-logo {
+  width: 130.69px;
+  height: 43.31px;
+}
+
+.login-register {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 34px;
+}
+
+.button-register {
+  padding: 5px 40px;
+  border: 1px solid var(--black);
+  border-radius: 5px;
+  background-color: var(--white);
+
+  font-size: 16px;
+  font-weight: 500;
+
+  cursor: pointer;
+}
+
+.banner-home {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 142px;
+}
+
+.banner-images {
+  display: flex;
+  gap: 20px;
+}
+
+.events-gifts {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.catering-merchan {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.banner-image1 {
+  width: 222.69px;
+  height: 192px;
+}
+
+.banner-image2 {
+  width: 222.62px;
+  height: 270.73px;
+}
+
+.banner-image3 {
+  width: 222.62px;
+  height: 231.39px;
+}
+
+.banner-image4 {
+  width: 222.95px;
+  height: 231.51px;
+}
+
+.banner-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 60px;
+}
+
+.banner-header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 20px;
+
+  width: 588px;
+  height: 195px;
+}
+
+.logo-companies {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.logo {
+  width: 17px;
+  height: 25.02px;
+}
+
+.banner-header h2 {
+  font-weight: 600;
+  font-size: 60px;
+  line-height: 108%;
+  letter-spacing: -1px;
+  background: var(--primary-gradient);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.banner-header h2 span {
+  -webkit-text-fill-color: var(--black);
+}
+
+.banner-buttons {
+  display: flex;
+  gap: 21px;
+  margin-top: 60px;
+}
+
+.register-here {
+  width: 225px;
+  height: 36px;
+  border-radius: 5px;
+  border: 0px;
+  background: var(--primary-gradient);
+
+  color: var(--white);
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.25);
+
+  text-transform: uppercase;
+  cursor: pointer;
+}
```

Comment:
Let's use padding for styling buttons instead of width and height. It is normally better to use padding since the button will support different text lengths and let the padding control the width and height depending on the text within the button.